packages/terra-date-picker/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Removed onInputFocus and onCalendarButtonClick props and pass them down as a custom props.
```

Comment:
update to include you are removing shouldShowPicker as well from the props

Reply:
Updated. https://github.com/cerner/terra-core/pull/1020/commits/7a9eb3f4af2b097fcd2625d84783b5bf93023666